### ISSUE TITLE
Review: abnormalPigmentation

### ISSUE THREAD

**Issue by srobb1** [obophenotype/upheno#349]
[abnormalPigmentation.yaml](https://github.com/obophenotype/upheno/blob/master/src/patterns/abnormalPigmentation.yaml)

If I am understanding the concerns of @sbello here in PR:https://github.com/obophenotype/upheno/pull/339 I think this pattern should be renamed to 
 - abnormalPigmentationInLocation or 
 - abnormalPigmenationInAnatomicalEntity or 
 - abnomalPigmentationInOrgan (if restricted to organ).

The same would apply to
 - [increasedPigmentation.yaml](https://github.com/obophenotype/upheno/blob/master/src/patterns/increasedPigmentation.yaml)
  - [decreasedPigmentaion.yaml](https://github.com/obophenotype/upheno/blob/master/src/patterns/decreasedPigmentation.yaml)

**Comment by sbello** [comment]
We're going to want either the first or second option as the entity is often smaller/more specific than an entire organ.

**Comment by srobb1** [comment]
@sbello Sounds good to me! I just created a PR https://github.com/obophenotype/upheno/pull/351 with the changes.

**Comment by matentzn** [comment]
I do think we will need the abnormal pigmentation process variants eventually, but for now this looks good. I am slowly beginning to think that I would like patterns to come with a "decreased", "increased" modification option to stop this huge proliferation of patterns; But lets roll with it for now.

**Comment by nicolevasilevsky** [comment]
@matentzn yes, it would be so nice if the patterns could include abnormal, increased and decreased and we wouldn't need separate patterns for each of those.

**Comment by nicolevasilevsky** [comment]
pattern_name: abnormalPigmentationOfAnatomicalEntity
pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormalPigmentationAnatomicalEntity
description: "Any change in the amount of pigmentation in an anatomical Entity. i.e. Abnormal pigmentation of the epidermis"

classes:
  abnormal: PATO:0000460
  anatomical entity: UBERON:0001062 
  degree of pigmentation: PATO:0002247

relations: 
  inheres_in: RO:0000052
  has_modifier: RO:0002573
  has_part: BFO:0000051

annotationProperties:
  exact_synonym: oio:hasExactSynonym 

vars:
  anatomical_entity: "'anatomical entity'"

name:
  text: "abnormal degree of pigmentation of the %s"
  vars:
   - anatomical_entity

annotations:
  - 
    annotationProperty: exact_synonym
    text: "abnormal pigmentation of the %s"
    vars:
     - anatomical_entity     

def:
  text: "abnormal degree of pigmentation of the %s."
  vars:
    - anatomical_entity

equivalentTo:
  text: "'has_part' some ('degree of pigmentation' and ('inheres_in' some %s) and ('has_modifier' some 'abnormal'))"
  vars:
    - anatomical_entity


@obophenotype/phenotype-editors could you all review this pattern above and sign off on it in the [spreadsheet](https://docs.google.com/spreadsheets/d/1KRx7aWd53PaPEmY-xDAurcNsfHP2EFTbY-J6LzP_dvk/edit#gid=442249367).
Nico hopes to have these all reviewed by mid-March, in time for the workshop at Biocuration2019.

**Comment by chris-grove** [comment]
Can we name this pattern "abnormalPigmentationAmountOfAnatomicalEntity" or something? I would imagine that a pattern called "abnormalPigmentationOfAnatomicalEntity" could apply in cases where the quality of the pigmentation changes (color, hue, pattern, etc.) but the degree/amount isn't necessarily affected. So, can we either change the pattern label, or broaden the meaning, or make separate patterns for general pigmentation abnormalities and degree/amount abnormalities specifically?
